src/content/docs/ecosystem/governance/overview.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Governance
+description: Overview of Governance on SKALE
+---
+
+
+import { Aside } from '@astrojs/starlight/components';
+
+<Aside type="note">
+SKALE DAO and Governance is early in its launch process and this document may change frequently as feedback is captured.  
+</Aside>
+
+SKALE Network is a network of Layer-1 blockchains with a root security anchor in Ethereum.
+The distribution and staking of the the SKL utility token enables decentralized governance of the SKALE Network.
+Delegated and staked SKL tokens enable voting on SKALE DAO governance proposals, thereby facilitating SKL delegators to collaboratively guide the evolution of SKALE.
+
+## A Refresher on SKALE Network
+
+SKALE Network is a high-performance, cost-effective blockchain network. Developers utilize SKALE to build user-friendly and
+performant decentralized applications (dApps) benefiting from the scalability offered by the SKALE Network.
+
+Launched in 2020, the SKALE Network has continuously improved its performance and scalability. The SKL utility tokens facilitate
+decentralized control over the SKALE Network and its underlying infrastructure. Staked SKL tokens permit voting on SKALE DAO
+governance proposals, enabling SKL token holders who stake to influence the future of SKALE collectively.
+
+## What is governance?
+
+Governance refers to decision-making processes and actions by which systems update and maintain themselves. Traditional governance
+in companies, governments, and institutions involves a decision making process that is relatively centralized and permissioned - for
+example processes that are made by a board of directors elected by shareholders.
+
+Web3 technologies, like SKALE’s protocols and chains, are initially developed by companies but progressively become decentralized with
+the growth of a user and stakeholder community. This progressive decentralization typically involves DAO formation and launching a
+process by which certain stake holders may vote. In this way, the SKALE DAO, a decentralized autonomous organization, has
+decision-making authority over the SKALE Network and its associated technologies. To steer the SKALE DAO, staking SKL tokens signifies
+DAO membership. Staked token holders can vote on DAO proposals.
+
+## How can I participate?
+
+- [Delegate to a SKALE validator](/ecosystem/delegations/overview).
```

Comment:
This link is incorrect. It should redirect to one of the two:
- ecosystem/delegators/overview: this is the correct path of the delegators overview page
- the portal for perform delegations: https://portal.skale.space/staking

Reply:
Agree -- will fix

Reply:
Resolved by [527cf3e](https://github.com/skalenetwork/docs.skale.space/pull/127/commits/527cf3ef3f69cd2ecfbbc70e85576610d7ee2377)